src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -1,24 +1,94 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdle.LEDStripType;
+import com.ctre.phoenix.led.CANdleConfiguration;
+import com.ctre.phoenix.led.RainbowAnimation;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.RobotMap;
+import java.util.TreeMap;
 
 public class Lights extends SubsystemBase {
+  private TreeMap<LightCode, Integer[]> lightOptionsMap;
+  private CANdle candle = new CANdle(RobotMap.CANDLE_CAN_ID);
+  private CANdleConfiguration config = new CANdleConfiguration();
+  private LightCode currentLightStatus = LightCode.OFF;
+  
+  /** Timer for the Blinking LED */
+  private int blinkingTimer = 0;
+  /** When the blinkingTimer reaches 10 (blinkingPeriod), the LED is toggled */
+  private int blinkingPeriod = 10;
+  /** If LEDs are currently blinking, then True would mean color is currently showing */
+  private boolean blinkingColorOn = false;
+
+
   public enum LightCode {
     GAME_OBJECT, // green
     READY_TO_SCORE, // blue
-    NO_TAG, // red
+    NO_TAG, // green larson animation
```

Comment:
make sure to change the comment to accurately reflect the blinking you actually program